tensorzero_internal/src/endpoints/openai_compatible.rs

```diff
@@ -228,17 +228,19 @@ impl TryFrom<(HeaderMap, OpenAICompatibleParams)> for Params {
     ) -> Result<Self, Self::Error> {
         let function_name = openai_compatible_params
             .model
-            .strip_prefix("tensorzero::")
+            .strip_prefix("tensorzero::function_name::")
+            // Allow 'tensorzero::' for backwards compatibility
+            .or_else(|| openai_compatible_params.model.strip_prefix("tensorzero::"))
```

Comment:
Let's print a warning here if someone uses the old format:

```
Deprecation Warning: Please set the `model` parameter to `tensorzero::function_name::your_function` instead of `tensorzero::your_function.` The latter will be removed in a future release.
```